### PR TITLE
improvements to Windows build & test situation

### PR DESCRIPTION
**Pull request by SethTisue**
review by @lrytz, @adriaanm, @som-snytt

this represents major progress towards: https://github.com/scala/scala-jenkins-infra/issues/36

a test run of the combination of this and: https://github.com/scala/scala-jenkins-infra/pull/132
is in progress at:
  ~~https://scala-ci.typesafe.com/job/scala-2.11.x-integrate-windows/53/~~
  https://scala-ci.typesafe.com/job/scala-2.11.x-integrate-windows/55/

### CODE REVIEW

File: scripts/jobs/integrate/windows [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+
+./pull-binary-libs.sh
+
+export ANT_OPTS="-Dfile.encoding=UTF-8 -server -XX:+AggressiveOpts -XX:+UseParNewGC -Xmx2G -Xss1M -XX:MaxPermSize=512M -XX:ReservedCodeCacheSize=128M"
+
+/home/tisue/apache-ant-1.9.6/bin/ant.bat \
```
Comment (adriaanm) [inline scripts/jobs/integrate/windows]: Abstract over username? :)

File: test/partest [hunk 2]
```diff
@@ -22,6 +22,14 @@ findScalaHome () {
...
 # Use tput to detect color-capable terminal.
+# (note: I have found that on Cygwin, the script sometimes dies here.
+# it doesn't happen from the Cygwin prompt when ssh'ing in to
+# jenkins-worker-windows-publish, only when I make a Jenkins job
+# that runs this script.  I don't know why.  it may have to do with
+# which bash flags are set (-e? -x?) and with bash flags propagating
+# from one script to another? not sure. anyway, normally in a CI
+# context we run partest through ant, not through this script, so I'm
+# not investigating further for now.)
 term_colors=$(tput colors 2>/dev/null)
```
Comment (adriaanm) [inline test/partest:33]: There's a magic incantation to make stuff always succeed. I believe it's `||:`

File: scripts/jobs/integrate/windows (hunk 1 above)
Comment (SethTisue) [inline scripts/jobs/integrate/windows]: pushed an extra commit that makes this somewhat less gross, but only somewhat. it's commented TODO. let's discuss next week

**Comment by som-snytt** [comment]
Is `tisue` just a typo for `suite`?

**Comment by lrytz** [comment]
LGTM, i'll merge this, the script can be refined in the future if necessary.

**Pull request merged**